kd/deployment.yaml

```
@@ -0,0 +1,243 @@
+            - --domain=hocs-case-creator.${KUBE_NAMESPACE}.svc.cluster.local
+            - --domain=localhost
+            - --onetime=true
+          env:
+            - name: KUBE_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - name: certs
+              mountPath: /certs
+            - name: keystore
+              mountPath: /etc/keystore
+            - name: bundle
+              mountPath: /etc/ssl/certs
+              readOnly: true
+          resources:
+            limits:
+              memory: 64Mi
+              cpu: 900m
+            requests:
+              memory: 16Mi
+              cpu: 300m
+
+      containers:
+        - name: certs
+          image: quay.io/ukhomeofficedigital/cfssl-sidekick:v0.0.6
+          securityContext:
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - SETUID
+                - SETGID
+          args:
+            - --certs=/certs
+            - --domain=hocs-case-creator.${KUBE_NAMESPACE}.svc.cluster.local
+            - --expiry=8760h
+            - --command=/usr/local/scripts/trigger_nginx_reload.sh
+          env:
+            - name: KUBE_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - name: certs
+              mountPath: /certs
+            - name: bundle
+              mountPath: /etc/ssl/certs
+              readOnly: true
+          resources:
+            limits:
+              memory: 32Mi
+              cpu: 400m
+            requests:
+              memory: 8Mi
+              cpu: 100m
+
+        - name: proxy
+          image: quay.io/ukhomeofficedigital/nginx-proxy:v3.4.12
+          imagePullPolicy: Always
+          securityContext:
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - SETUID
+                - SETGID
+          env:
+            - name: PROXY_SERVICE_HOST
+              value: '127.0.0.1'
+            - name: PROXY_SERVICE_PORT
+              value: '8080'
+            - name: NAXSI_USE_DEFAULT_RULES
+              value: 'FALSE'
+            - name: ENABLE_UUID_PARAM
+              value: 'FALSE'
+            - name: HTTPS_REDIRECT
+              value: 'FALSE'
+            - name: BASIC_AUTH
+              value: /etc/nginx/authsecrets/htpasswd
+            - name: SERVER_CERT
+              value: /certs/tls.pem
+            - name: SERVER_KEY
+              value: /certs/tls-key.pem
+            - name: ADD_NGINX_SERVER_CFG
+              value: >-
+              location = /reload { allow 127.0.0.1; deny all; content_by_lua_block
+              { os.execute("touch /tmp/nginx-reload-triggered;
+              /usr/local/openresty/nginx/sbin/nginx -s reload;
+              touch /tmp/nginx-reload-complete;") } }
```

This needs to be indented by two lines:
```suggestion
              value: >-
                location = /reload { allow 127.0.0.1; deny all; content_by_lua_block
                { os.execute("touch /tmp/nginx-reload-triggered;
                /usr/local/openresty/nginx/sbin/nginx -s reload;
                touch /tmp/nginx-reload-complete;") } }
```
but you may as well indent it sensibly while we're here:
```suggestion
              value: >
                location = /reload {
                  allow 127.0.0.1;
                  deny all;
                  content_by_lua_block {
                    os.execute("touch /tmp/nginx-reload-triggered;
                      /usr/local/openresty/nginx/sbin/nginx -s reload;
                      touch /tmp/nginx-reload-complete;")
                    }
                 }
```